bundles/org.eclipse.passage.lic.jface/src/org/eclipse/passage/lic/jface/dialogs/licensing/StatusFromException.java

```diff
@@ -0,0 +1,60 @@
+/*******************************************************************************
+ * Copyright (c) 2020 ArSysOp
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * https://www.eclipse.org/legal/epl-2.0/.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     ArSysOp - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.passage.lic.jface.dialogs.licensing;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Supplier;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.eclipse.core.runtime.IStatus;
+import org.eclipse.core.runtime.MultiStatus;
+import org.eclipse.core.runtime.Status;
+import org.eclipse.passage.lic.internal.api.diagnostic.Diagnostic;
+
+public final class StatusFromException implements Supplier<IStatus> {
```

Comment:
Please move to lic.equinox

Reply:
Oops, we face an accident!

Reply:
All the work has been moved to `internal` package